integration/tests/inferentia/inferentia_test.go

```diff
@@ -170,18 +172,18 @@ var _ = Describe("(Integration) Inferentia nodes", func() {
 					Expect(cmd).To(RunSuccessfully())
 				})
 				It("should install the neuron device plugin", func() {
-					_, err := newClientSet().AppsV1().DaemonSets("kube-system").Get(context.TODO(), "neuron-device-plugin-daemonset", metav1.GetOptions{})
+					_, err := newClientSet(clusterName).AppsV1().DaemonSets("kube-system").Get(context.TODO(), "neuron-device-plugin-daemonset", metav1.GetOptions{})
 					Expect(err).ShouldNot(HaveOccurred())
 				})
 			})
 		})
 	})
 })
 
-func newClientSet() *kubernetes.Clientset {
+func newClientSet(name string) *kubernetes.Clientset {
 	cfg := &api.ClusterConfig{
 		Metadata: &api.ClusterMeta{
-			Name:   defaultCluster,
+			Name:   name,
 			Region: params.Region,
 		},
 	}
```

Comment:
how does this differ to whats currently being done? Both `params` and `defaultCluster` are variables defined in the suite, and populated in the `init`. I feel like I'm missing something

Reply:
there are more than one cluster created:
https://github.com/weaveworks/eksctl/pull/3259/files#diff-089af4f1a66bd9a4c22a730918fe1810f044e1b910ec017506773e58f9ac59d9R117

Reply:
ahhhh 👍